doc/howto/write_a_view/index.md

```diff
@@ -9,7 +9,7 @@ This HowTo documents how to write a view using the standard library and some hel
 \note
 Some of the links from this HowTo only resolve in the developer documentation because
 they refer to entities from the seqan3::detail namespace.
-We recommend you open this tutorial from [the developer documentation](http://docs.seqan.de/seqan/3.0.0-master-dev/).
+We recommend you open this tutorial from [the developer documentation](https://docs.seqan.de/seqan/3.0.0-master-dev/).
```

Comment:
I think it is https://docs.seqan.de/seqan/3-master-dev/

Reply:
Good catch. I'll check if there are more occurrences